src/hivpy/sexual_behaviour.py

```diff
@@ -158,166 +334,213 @@ def num_short_term_partners(self, population: Population):
+    def update_risk_long_term_partnered(self, pop: Population):
+        pop.set_present_variable(col.RISK_LTP, 1)  # Unpartnered people
         partnered_pop = pop.get_sub_pop([(col.LONG_TERM_PARTNER, operator.eq, True)])
-        pop.set_present_variable(col.RRED_LTP, self.ltp_risk_factor, partnered_pop)
+        pop.set_present_variable(col.RISK_LTP, self.ltp_risk_factor, partnered_pop)
 
-    def init_rred_personal(self, population: Population):
-        population.init_variable(col.RRED_PERSONAL, 1)  # personal rred doesn't update(?)
+    def init_risk_personal(self, population: Population):
+        population.init_variable(col.RISK_PERSONAL, 1)  # personal risk doesn't update(?)
         r = rng.uniform(size=population.size)
-        mask = r < self.p_rred_p
-        population.set_present_variable(col.RRED_PERSONAL, 1e-5, mask)
-
-    def init_rred_adc(self, population: Population):
-        population.init_variable(col.RRED_ADC, 1.0)
-        self.update_rred_adc(population)
-
-    def update_rred_adc(self, population: Population):
+        mask = r < self.p_risk_p
+        population.set_present_variable(col.RISK_PERSONAL, 1e-5, mask)
+
+        females = population.get_sub_pop([(col.SEX, operator.eq, SexType.Female)])
+        population.set_present_variable(col.LIFE_SEX_RISK, 2, females)
+        low_risk_females = population.get_sub_pop_intersection(females, population.apply_bool_mask(mask))
+        population.set_present_variable(col.LIFE_SEX_RISK, 1, low_risk_females)
+        mask = r > (1 - self.prob_high_sex_risk)
+        high_risk_females = population.get_sub_pop_intersection(females, population.apply_bool_mask(mask))
+        population.set_present_variable(col.LIFE_SEX_RISK, 3, high_risk_females)
+
+    def init_risk_adc(self, population: Population):
+        population.init_variable(col.RISK_ADC, 1.0)
+        self.update_risk_adc(population)
+
+    def update_risk_adc(self, population: Population):
         """
         Updates risk reduction for AIDS defining condition
         """
-        # We don't need the rred_adc==1 condition (since they are all set to rred_adc = 1 to start)
+        # We don't need the risk_adc==1 condition (since they are all set to risk_adc = 1 to start)
         # It prevents needless assignments but requires checking more conditions
         # Not sure which is more efficient or if it matters.
         indices = population.get_sub_pop([(col.ADC, operator.eq, True),
                                           (col.HIV_STATUS, operator.eq, True)])
-        population.set_present_variable(col.RRED_ADC, 0.2, indices)
+        population.set_present_variable(col.RISK_ADC, 0.2, indices)
 
-    def init_rred_population(self):
+    def init_risk_population(self):
         """
-        Initialise general population risk reduction w.r.t. condomless sex with new partners
+        Initialise general population risk reduction w.r.t. condomless sex withnew partners
```

Comment:
```suggestion
        Initialise general population risk reduction w.r.t. condomless sex with new partners
```
Accidental space removal?